src/app/[locale]/(routes)/settings/page.tsx

```diff
@@ -1,17 +1,17 @@
-'use client'
-import { FC } from "react";
-import { useTranslations } from "next-intl";
-interface pageProps {
-    
-}
- 
-const page: FC<pageProps> = () => {
-   const t = useTranslations('HelpCenter');
-    return ( <>
-    <section>
-        <div>{t('title')}</div>
-   </section>
-    </> );
-}
- 
-export default page;
\ No newline at end of file
+'use client';
+import { FC } from 'react';
+import { useTranslations } from 'next-intl';
+interface pageProps {}
+
+const Page: FC<pageProps> = () => {
+	const t = useTranslations('HelpCenter');
+	return (
+		<>
+			<section>
+				<div>{t('title')}</div>
+			</section>
+		</>
+	);
+};
```

Comment:
What happen with the indentation? 
Check your prettier config

Reply:
done